src/main/content/try-it.html

```diff
@@ -69,19 +79,22 @@
             <div class="tab-pane fade show" id="download_appsody" role="tabpanel" aria-labelledby="download_appsody">
                 <div class="row">
                     <div class="col download_detail">
-                        Coming soon!
+                        <p>
+                            Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications.
+                        </p>
+                        <a href="https://appsody.dev">View the Appsody and Appsody for Codewind download and install instructions</a>
```

Comment:
add the `target="_blank"` attribute  for this link and the codewind.dev link above too